alerts/utils.go

```diff
@@ -29,6 +34,32 @@ func (alert *Alert) setStatus(status bool){
 	alert.State = status
 }
 
-func (Alerts Alerts) Open() {}
+func (a *Alerts) Open() {
+	// start a maintenance worker that will check for alerts periodically.
+	go a.maintenance()
+}
+
+func (a *Alerts) maintenance() {
+	// clean data every 120 minutes
+	c := time.Tick(120 * time.Minute)
+
+	// once a tick check for alerts in data
+	for range c {
+		fmt.Printf("maintenance: start\n")
+		a.checkAlerts()
+	}
+}
+
+func (a *Alerts) checkAlerts() {
+	for _, alert_list := range a.AlertLists {
+		// Get each tenants item list
+		for _, alert := range alert_list.List {
+			raw_data := a.Backend.GetRawData(alert_list.Tenant.Id ,alert.Id,80,0, 0,"ASC")
```

Comment:
I am not sure what method to call here to get the last value / last X values.
cc: @yaacov

Reply:
if it works it is good enough for now, re-factor after you tested it and you think you know what needs to be re-factored and what is not.

Reply:
@yaacov I'm not sure what values I should put at `start` & `end` for the `GetRawData`??

Reply:
https://github.com/MohawkTSDB/mohawk/blob/master/backend/backend.go#L59

`start` and `end` are javascript timestamp [1] in milisec 
limit is the number of samples you want
order is DESC of ASC string

like an sql query:
`select from reading when time > start and time < and sort by time ASC limit 1`

[1] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now

Reply:
@yaacov hmm but I dont care about `end` and `start` and timestamps.. I just want the last time/value pair inserted. Isn't there a simple method for getting this data?

Reply:
you can try `GetItemList` but it has other drawbacks.
or you can implement a new callback if you want in all backends ...